### DISCUSSION TITLE
ChromeOS linux environment

### DISCUSSION THREAD

**Discussion by mderazon** [ghostty-org/ghostty#3409] [Q&A] [answered]
Has anyone been able to run Ghostty in ChromeOS linux environment ?

**Comment by keidax** [comment]
Yep, after building from source!

Overall Ghostty seems to run okay, except I can't open the main menu -- the popup corrupts and doesn't display fully. 

The default container for ChromeOS Linux uses Debian 12, which currently packages `libgtk-4-dev` at version `4.8.3`. I'm hoping that building Ghostty with a newer version of GTK4 might fix this.

**Reply by juan-m-medina** [reply]
I got it to start, but when I open it the prompt/shell keep looping/blinking as if the shell was being reinitialized in a loop. Input does not work. I did try solutions to a maybe related discussion https://github.com/ghostty-org/ghostty/discussions/3224 but no luck.

**Reply by keidax** [reply]
Just a wild guess, have you enabled GPU support? chrome://flags/#crostini-gpu-support

**Reply by rhodes-b** [reply]
Yeah GTK 4.8 is really old and most likely doesnt work in ghostty

I believe you need at minimum 4.10 for it to work properly I know that 4.14 as the minimum should work for sure though

**Reply by keidax** [reply, marked as answer]
I set up a new Crostini container with Debian testing ("trixie"). After building with `-fsys=fontconfig` (see https://github.com/ghostty-org/ghostty/discussions/3311) ghostty seems to be working well!
